README.md

```diff
@@ -1,8 +1,68 @@
+# JSON Compilation Database Format Specification
+
+This document describes a format for specifying how to replay single link commands independently of the build system.
+
+## Background
+Currently, there is compilation database, which specified by [Clang](https://clang.llvm.org/docs/JSONCompilationDatabase.html). Compilation commands are actively used for syntax analysis of code, 
```

Comment:
Link should be for compilation database, not clang. And compilation database -> compilation database format

Reply:
+ an article: there is a compilation database format